project.yaml

```diff
@@ -4,15 +4,8 @@ expectations:
   population_size: 1000
```

Comment:
This is no longer relevant with ehrQL and therefore a source of confusion to have it here. But currently the pipeline validator insists that this be defined for version 3.0 project.yaml files:
https://github.com/opensafely-core/pipeline/blob/8c706d663fb2fcae279bc0f15c1b1b3bd4314e99/pipeline/models.py#L186-L211

So I think we're going to have to move to version 4 for project.yaml files and create a new release of the pipeline library, which is incredibly tedious.

For now, I wonder if we should just stick a comment above this expectations section saying something like "Please ignore this section, it is not used and will be removed in future versions."

Reply:
Have created an issue to track this:
 * https://github.com/opensafely-core/pipeline/issues/224

Reply:
Thanks!